utils/netns/netns.go

```diff
@@ -59,3 +61,32 @@ func IsVethExists(vethHostName string) (bool, error) {
 	}
 	return false, nil
 }
+
+// GetLinkLocalAddr Construct the IPv6 link local address of interfaceName
+// from the interface MAC address.
+func GetLinkLocalAddr(interfaceName string) net.IP {
+	link, e := netlink.LinkByName(interfaceName)
+	if e != nil {
+		log.Errorf("GetLinkLocalAddr: %v", e)
+		return nil
+	}
+	mac := link.Attrs().HardwareAddr
```

Comment:
I don't think the github.com/vishvananda/netlink package retrieves the IP address we want. At least I didn't see it anywhere in `link.Attrs().*`. I changed it up to use "ip addr" similar to the python version.